core/templates/pages/exploration-editor-page/translation-tab/audio-translation-bar/audio-translation-bar.directive.ts

```diff
@@ -554,9 +554,8 @@ angular.module('oppia').directive('audioTranslationBar', [
               $scope.audioBlob = null;
             });
 
-            $scope.$on('activeContentIdChanged', function() {
-              $scope.initAudioBar();
-            });
+            TranslationTabActiveContentIdService.onActiveContentIdChange
+              .subscribe(() => $scope.initAudioBar());
```

Comment:
Wrap this thing in brackets
```suggestion
            (TranslationTabActiveContentIdService.onActiveContentIdChange
              .subscribe(() => $scope.initAudioBar()));
```

Reply:
Similarly for the other subscribe as well.

Reply:
This will go in subscription instance